### ISSUE TITLE
Tag taxonomy project: Q2, tag analysis and deriving a controlled vocab

### ISSUE THREAD

**Issue by VanCardui** [SingularityNET-Archive/SingularityNET-Archive#133]
Follow-on from the work done in Q1 here https://github.com/SingularityNET-Archive/SingularityNET-Archive/issues/87 https://github.com/SingularityNET-Archive/SingularityNET-Archive/issues/144 https://github.com/SingularityNET-Archive/SingularityNET-Archive/issues/145 and https://github.com/SingularityNET-Archive/SingularityNET-Archive/issues/146.

This Issue is for the work to analyse our tags and see if it's possible to create a controlled vocabulary for any elements of it. This is costed in Quarter 2, and the planned completion date is the end of June

See workplan here https://docs.google.com/document/d/14q0GxN0c-QraeSt1Z-bINScd5LJQr275HTjN6C9lyHY/edit?usp=sharing,

**Comment by stephen-rowan** [comment]
In progress - updates to be completed last week of June 2024

**Comment by stephen-rowan** [comment]
Decision - by Archive WG - Discord ID = Archive ID - add this guidance to front page of Archive GitBook

**Comment by VanCardui** [comment]
RE: "**_Discord ID = Archive ID - add this guidance to front page of Archive GitBook_**": I added

"_Note that in meeting summaries, "Discord name= Archive name": that is, we record individual people's involvement in meetings using their Discord names as far as we can. So if you want to search or query the Archives about a particular person, it's best to use their Discord name_."

**Comment by stephen-rowan** [comment]
Complete async in August 2024

**Comment by stephen-rowan** [comment]
Suggestion - We add review of new tags to Archive WG meetings
People will be able to add tags but won't be visible until approved.

**Comment by VanCardui** [comment]
in 8th Jan meeting we discussed the composability of our data - could a tag taxonomy sit on top of the data as another layer? Similarities with the overall graph approach

We need to see how our current tags actually work with a graph rag approach - maybe there is less need than we thought to create a strong controlled vocab and e.g. clear out misspellings and duplicates regularly

**Comment by VanCardui** [comment]
Agreed in meeting 5th Feb that the work we thought we needed to do (i.e. exhaustively de-duping tags in the database itself), we might not need to do
But we probably should define a controlled vocab for topic tags for the future., just stripping out obvious misspellings, duplicates with different capitalisation,  tags that are too generic ("budget" rather than "Q2 2024 budget") etc, which can be added to the summary tool - but for the rest it will be interesting to see if semantic similarity searches can find tags with similar meanings, particularly the "emotion" tags. We will continue to leave open the possibility of creating new tags; and the subtle differences, especially on emotions, may be important.

André and Vani to implement these controlled vocabs, then this issue can be closed

**Comment by VanCardui** [comment]
Finalising controlled vocabs for the different tag fields here https://docs.google.com/spreadsheets/d/1N73wvHjOO26EUuqNAltbuNS1VGF3QmvrA8vUIpZ8tF8/edit?usp=sharing
